config/govuk_index/mapped_document_types.yaml

```diff
@@ -25,6 +25,7 @@ mainstream_browse_page: edition # Collections Publisher
 manual: manual
 manual_section: manual_section
 medical_safety_alert: medical_safety_alert # Specialist Publisher
+news_story: edition
```

Comment:
As we're not too sure on the why or the implications could we put this in a separate commit to document the uncertainty?

Reply:
Hopefully this commit explains this decision: https://github.com/alphagov/rummager/pull/1276/commits/3983732cc5e71b8771fdd09eab045867528c148a